templates/page.html

```diff
@@ -58,7 +58,7 @@
     <a href="http://www.sonarqube.org/"><img src="${rellink('images/sonarqube.png')}" height="40" width="140" alt="Inspected with sonarqube" style="border:0px solid black"/></a>
   </div>
   <div style="text-align:center;margin-top:10pt;padding-top:15px;border-top:1px solid #a0a0a0">
-    <a href="http://www.mtrail.de/jobs"><img src="${rellink('images/mtrail.png')}" width="140" height="200" alt="Java Developer in Frankfurt" style="border:0px solid black"/></a>
+    <a href="https://www.mtrail.ch/de/karriere.html"><img src="${rellink('images/mtrail.png')}" width="140" height="200" alt="Java Developer in Frankfurt" style="border:0px solid black"/></a>

```

Comment:
Few minor suggestions - IMO will be better to
* use English version of the page instead of German
* use more generic alt text, because linked page contains not only "for Frankfurt"

```suggestion
    <a href="https://www.mtrail.ch/en/career.html"><img src="${rellink('images/mtrail.png')}" width="140" height="200" alt="jobs at mtrail" style="border:0px solid black"/></a>
```